README.md

```diff
@@ -0,0 +1,95 @@
+<p align="center">
+	<a href="https://github.com/doximity/CallWithDoxDialer/"><img src="ReadmeResources/logo.png" width="200" alt="CallWithDoxDialer" /></a><br /><br />
+	A µLibrary for making calls using <a href="https://www.doximity.com/clinicians/download/dialer/">Doximity Dialer</a>.<br /><br />
+</p>
+<br />
+
+[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](#carthage) [![CocoaPods compatible](https://img.shields.io/cocoapods/v/CallWithDoxDialer.svg)](#cocoapods) [![GitHub release](https://img.shields.io/github/release/doximity/CallWithDoxDialer.svg)](https://github.com/doximity/CallWithDoxDialer/releases) ![platforms](https://img.shields.io/badge/platforms-iOS-lightgrey.svg)
+
+
+## What is CallWithDoxDialer?
+
+Doximity's [Dialer][] app lets healthcare professionals make
```

Comment:
oh that's already a link, that's why the format is weird.
The associated link is at the bottom of the file